ocs_ci/ocs/elasticsearch.py

```diff
@@ -42,14 +42,10 @@ def elasticsearch_load(connection, target_path):
         log.error("There is No data to load into ES server")
         return False
     else:
-        log.info(f"Loading data from {target_path} to ES server at {connection}")
-        try:
-            log.info(f"Creating new ES connection to {connection}")
-            main_es = Elasticsearch([connection])
-            log.debug(f"Connection info : {main_es}")
-        except esexp.ConnectionError:
-            log.error(f"Can not connect to the main ES server on {connection}")
+        if connection is None:
```

Comment:
the best I thing is add a conf item to disable it along with cli option.

Reply:
There are Parameters in the test CRD file (for FIO and SmallFiles) the the elasticsearch information.
if this section in the CRD file is omitted, it will not try to create the elasticsearch object,
so, i don't think we need another parameter for enable/disable the elasticsearch